### DISCUSSION TITLE
Impossible to start the program :(

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/OdfEdit#64] [General]
Dear @eturpault, I've downloaded Odfedit from your GitHub, but it does not start .. I've put in the Properties "run as a program" but when I double click on the icon, nothing happens. 

I'm on Linux Mint 21.3 base Ubuntu 22.04

Many thanks :-)

**Comment by eturpault** [comment]
Dear @Jean-Gabriel1982, before to release a new OdfEdit version I check that it is running well in Windows and Linux. So it is not expected that it is not running in your side. I am using Linux Ubuntu 22.04.2 LTS in VirtualBox.
With these settings OdfEdit is running well in my environment :
![image](https://github.com/user-attachments/assets/2d9ef78c-23b8-4d47-b06e-e66c63f65b3b)

I don't know well Linux, maybe someone else in this forum knowing better Linux could be able to support you.
Else you can run OdfEdit using the Python framework as indicated in the ReadMe here : https://github.com/GrandOrgue/OdfEdit
